### PR TITLE
Env concept

### PR DESCRIPTION
**Pull request by alex-schneider**
Review #97 first!

### CODE REVIEW

File: src/schema.js [hunk 1]
```diff
@@ -4,53 +4,85 @@ const DEFAULT_LABEL = "…"
 
 const blocks = {
 	api: {
-		parents: ['server'],
+		parents: ['environment', 'server'],
```
Comment (afflerbach) [inline src/schema.js:7]: Unfortunately, this renders the completion almost useless. With `environment` as every blocks/attributes parent, the completion always suggests *all* blocks/attributes as possible children for `environment`:

broken_completion
ironment`.

**Review by malud — CHANGES_REQUESTED** [review]
Since all blocks are childs of `environment` the will be suggested on invalid places.

```hcl
environment "…" {
    server {
        environment "…" {
            endpoint "/" {
                proxy {
                    environment "…" {
                        oauth2 {
                            // invalid
                        }
                        access_control = [""] // invalid
                    }
                }
            }
        }
    }
}
```

**Review by afflerbach — CHANGES_REQUESTED** [review]
I guess we have to handle `environment` specially without adding it to the general schema.

**Review by johakoch — APPROVED** [review]
Approved on behalf of alex-schneider

**Review by malud — APPROVED** [review]

**Pull request merged**